examples/borer-circe-migration/build.sbt

```diff
@@ -0,0 +1,23 @@
+name := "borer-circe-migration"
+version := "0.1"
+scalaVersion := "2.13.6"
+
+idePackagePrefix := Some("pl.marconzet")
```

Comment:
Hmm maybe use `com.example` package instead... anyway, I assume https://marconzet.pl/ is your domain? ;)

Reply:
Or even `org.virtuslab.example`, usually I see that projects keep the examples in the similar packages as the actual production code